lib/rules/lint-no-curly-component-invocation.js

```diff
@@ -0,0 +1,98 @@
+const Rule = require('./base');
+const { transformTagName } = require('../helpers/curly-component-invocation');
+const IN_BUILT_HELPERS = [
```

Comment:
Mind changing this to `BUILT_IN_HELPERS`? I know its kinda pedantic, but its a bit easier for my mind to grok...